templates/sios-datakeeper-main.template.yaml

```diff
@@ -1,9 +1,9 @@
 AWSTemplateFormatVersion: '2010-09-09'
 Description: >-
-  This template deploys two Windows Server Failover Clustering (WSFC) nodes with SQL
+  This template deploys two or three Windows Server Failover Clustering (WSFC) nodes with SQL
   Server 2014 and DataKeeper replication software. This template also deploys the
```

Comment:
@davmayd Do we need to keep a year on "Server 2014"? I'd delete the year if we can for future-proofing.